libs/explorers/comparison-tool/src/lib/help-links/visualization-overview-panel/visualization-overview-panel.component.spec.ts

```diff
@@ -0,0 +1,297 @@
+import { provideNoopAnimations } from '@angular/platform-browser/animations';
+import { VisualizationOverviewPane } from '@sagebionetworks/explorers/models';
+import {
+  provideComparisonToolService,
+  ComparisonToolService,
+} from '@sagebionetworks/explorers/services';
+import { mockComparisonToolDataConfig } from '@sagebionetworks/explorers/testing';
+import { render, screen } from '@testing-library/angular';
+import userEvent from '@testing-library/user-event';
+import { VisualizationOverviewPanelComponent } from './visualization-overview-panel.component';
+
+describe('VisualizationOverviewPanelComponent', () => {
+  const mockPanes: VisualizationOverviewPane[] = [
+    {
+      heading: 'Welcome to Visualization',
+      content: '<p>First pane content</p>' as any,
+    },
+    {
+      heading: 'How to Use Filters',
+      content: '<p>Second pane content</p>' as any,
+    },
+    {
+      heading: 'Export Your Data',
+      content: '<p>Third pane content</p>' as any,
+    },
+  ];
+
+  async function setup(options?: {
+    isVisible?: boolean;
+    isHiddenByUser?: boolean;
+    panes?: VisualizationOverviewPane[];
+  }) {
+    const user = userEvent.setup();
+
+    const { fixture } = await render(VisualizationOverviewPanelComponent, {
+      providers: [
+        provideNoopAnimations(),
+        ...provideComparisonToolService({
+          configs: mockComparisonToolDataConfig,
+          viewConfig: {
+            selectorsWikiParams: {},
+            headerTitle: 'Test Title',
+            filterResultsButtonTooltip: 'Filter tooltip',
+            showSignificanceControls: false,
+            viewDetailsTooltip: 'View details',
+            viewDetailsClick: jest.fn(),
+            legendEnabled: false,
+            legendPanelConfig: {
+              colorChartLowerLabel: '',
+              colorChartUpperLabel: '',
+              colorChartText: '',
+              sizeChartLowerLabel: '',
+              sizeChartUpperLabel: '',
+              sizeChartText: '',
+            },
+            visualizationOverviewPanes: options?.panes ?? mockPanes,
+          },
+        }),
```

Comment:
What do you think about simplifying the `viewConfig` to only set options that are relevant to these tests? In other cases, we can rely on the defaults -- 

```suggestion
          viewConfig: {
            legendEnabled: false,
            visualizationOverviewPanes: options?.panes ?? mockPanes,
          },
        }),
```